src/actions/groups.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// @flow
+
+import {GroupTypes} from 'action_types';
+import {General, Groups} from 'constants';
+
+import {Client4} from 'client';
+
+import {logError} from './errors';
+import {bindClientFunc, forceLogoutIfNecessary} from './helpers';
+import {batchActions} from 'redux-batched-actions';
+
+import type {ActionFunc} from '../types/actions';
+import type {SyncableType, SyncablePatch} from '../types/groups';
+
+export function linkGroupSyncable(groupID: string, syncableID: string, syncableType: SyncableType, patch: SyncablePatch): ActionFunc {
+    return async (dispatch, getState) => {
+        dispatch({type: GroupTypes.LINK_GROUP_SYNCABLE_REQUEST, data: {groupID, syncableID, type: syncableType}});
+
+        let data;
+        try {
+            data = await Client4.linkGroupSyncable(groupID, syncableID, syncableType, patch);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: GroupTypes.LINK_GROUP_SYNCABLE_FAILURE, error, data: {groupID, syncableID, type: syncableType}},
+                logError(error),
+            ]));
+            return {error};
+        }
+
+        var type;
```

Comment:
0/5 but any reason to use `var` over `let` here?

Reply:
Also could ask the question about a few other places, but since it's 0/5 I'm not going to be pedantic - more just curious about the choice.